pkg/graphql/resolver/k8s.go

```diff
@@ -8,6 +8,7 @@ import (
 	"net/http"
 	"net/http/httptest"
 
+	authentication "k8s.io/api/authentication/v1"
 	auth "k8s.io/api/authorization/v1"
```

Comment:
nit, maybe use `authn` and `authz` convention here to make these less confusing?
```suggestion
	authn "k8s.io/api/authentication/v1"
	authz "k8s.io/api/authorization/v1"
```